deploy/charts/rook-ceph/values.yaml

```diff
@@ -63,7 +63,7 @@ csi:
   # Set to true to enable host networking for CSI CephFS and RBD nodeplugins. This may be necessary
   # in some network configurations where the SDN does not provide access to an external cluster or
   # there is significant drop in read/write performance.
-  # enableCSIHostNetwork: true
+  enableCSIHostNetwork: true
```

Comment:
Why you are uncommenting it, It's the user choice to turn it on

I think so we already can disable HostNetwork while making it false,

Are you taking abut helm